bolt-design-system/bolt
# Bolt v2.5.2 Hotfixes

## Jira
http://vjira2:8080/browse/BDS-1483
http://vjira2:8080/browse/BDS-1484
http://vjira2:8080/browse/BDS-1485

## Summary
Fixes a handful of build and DevOps-related bugs occurring when running a build outside of the Bolt monorepo. Also adds a new Drupal Lab build step to the default Travis CI jobs to now test installing and building outside of the monorepo-installed context. 

## Details
- fixes a broken package.json path (used for determining the current version of Bolt being run) when build tools are run outside of the monorepo
- fixed a separate NPM package version mismatch issue in `@bolt/core`'s polyfills -- this fix updates the filenames to a handful of polyfills used internally
- updated the `@bolt/build-tools`'s Webpack build to automatically add the `@bolt/core/styles/index.scss` file to the default global entrypoint. Addresses a strange behavior observed where the exported JSON data for Sass files in `@bolt/core` weren't being generated as expected when compiling outside of the Bolt monorepo.
- Adds a new Drupal Lab Travis job to now start to check for these unexpected build-related issues when running the codebase downstream

## How to test
- Confirm the Travis build passes
- Confirm the new Drupal Lab Travis CI job (which adds these 3 package patches) builds without any errors.
